### DISCUSSION TITLE
how do i install this video translator for youtube on android? (apk)

### DISCUSSION THREAD

**Discussion by test999s** [ilyhalight/voice-over-translation#235] [Q&A] [answered]
how do i install this video translator for youtube on android? (apk)

**Comment by miketype** [comment, marked as answer]
for youtube on android - install firefox or kiwi browser with [tempermonkey ](https://www.tampermonkey.net/) and this [extension](https://raw.githubusercontent.com/ilyhalight/voice-over-translation/master/dist/vot.user.js).
it a browser extension.

для ютуба на андроид - установите firefox или  kiwi browser с [tempermonkey ](https://www.tampermonkey.net/) и данным [расширением](https://raw.githubusercontent.com/ilyhalight/voice-over-translation/master/dist/vot.user.js).
это плагин для браузера.
